benchmark-service/benchmark/utils.py

```diff
@@ -45,6 +49,16 @@ def list_benchmark_files(read_gs):
     return list_of_files
 
 
+async def submit_batch(batch_client, sha):
+    batch = batch_client.create_batch()
+    job = batch.create_job(image='ubuntu:18.04',
+                           command=['/bin/bash', '-c', 'touch /io/test; sleep 300'],
```

Comment:
Can you make the resources on this be like this:

```
resources = {'cpu': 0.25}
```

We don't want to use entire cores for 5 minutes when we only have 3 cores available in the testing scenario.